components/eoy_artist_list/_artist_item.jade

```diff
@@ -1,6 +1,6 @@
 .eoy-artist-item
   .eoy-artist-item__thumbnail
-    a(href="/artist/#{artist.slug}"): img(src="#{sd.API_URL}/api/v1/artist/#{artist.slug}/image")
+    a(href="/artist/#{artist.slug}"): img(src="/search/image/artist/#{artist.slug}")
```

Comment:
lol, I guess that’s exactly what it does https://github.com/artsy/force/blob/a28b3867d7da64af5b7adfcd38c01f3f592222a7/apps/search/components/image_url/index.coffee#L7